.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,17 +1,30 @@
-# Описание
+## Краткое описание
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)